WPF/MainWindow.xaml.cs

```diff
@@ -300,6 +300,11 @@ private void Settings_Execute(object sender, ExecutedRoutedEventArgs e)
             StateSwitcher.Instance.OnProjectCreateOrEdit(model.GetProject());
             ProjectCreator pc = new ProjectCreator();
         }
+        
+        private void Account_Execute(object sender, ExecutedRoutedEventArgs e)
+        {
+            // For Makayla
```

Comment:
This should route through the state switcher, who will be in charge of handling the view if bad things happen. Ie. if a disconnect happens they close the account settings.